### DISCUSSION TITLE
Version 1.19.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#209] [Announcements]
- Web search shortcuts have been replaced with quick actions:
  - quick actions are shown below the search bar and are shortcuts that take your search term and do something with it
  - type a phone number to call somebody, an email address to write an email or "10 min" to start a 10 minute timer
  - you can also add you own actions to search on websites or in apps [^1] or to launch custom Android intents.
  - Existing web search shortcuts will be migrated automatically
- Minor improvements
- Bugfixes

[^1] Searching in apps is an experimental feature because it doesn't work reliably (or at all) in some apps. I'm still debating whether to keep this feature or not. Feedback/opinions on this are welcome.
  

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.19.0'>Version 1.19.0</a>.</em>

**Comment by TimoNotThy** [comment]
Searching in apps is amazing! How is it decided for what apps this is available? Do you check the intents?
